lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -26,11 +26,17 @@ def safe_add_enum_value(name, value)
   end
 
   def safe_add_column(table, column, type, options = {})
-    if options.has_key? :default
+    if options.has_key?(:default) && options[:default].include?("(")
       raise PgHaMigrations::UnsafeMigrationError.new(":default is NOT SAFE! Use safe_change_column_default afterwards then backfill the data to prevent locking the table")
     end
-    if options[:null] == false
-      raise PgHaMigrations::UnsafeMigrationError.new(":null => false is NOT SAFE if the table has data! If you _really_ want to do this, use unsafe_make_column_not_nullable")
+
+    unless ActiveRecord::Base.connection.postgresql_version >= 11_00_00
+      if options.has_key? :default
```

Comment:
```suggestion
      if options.has_key?(:default)
```

I know it wasn't this way before, but let's correct the style if we're going to change this code.